Example/Sources/Extensions/AlertService.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AlertService.swift
+//  ChatExample
+//
+//  Created by Mohannad on 12/25/20.
+//  Copyright © 2020 MessageKit. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+
+class AlertService {
+
+    
+    static func showAlert(style: UIAlertController.Style, title: String?, message: String?, actions: [UIAlertAction] = [UIAlertAction(title: "Ok", style: .cancel, handler: nil)], completion: (() -> Swift.Void)? = nil) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: style)
+        for action in actions {
+            alert.addAction(action)
+        }
+        if let topVC = UIApplication.getTopMostViewController() {
```

Comment:
If this doesn§t exist, which means sourceView is nil, doesn't this crash on iPad?

Reply:
I did an optional binding to prevent crashing,  and I've just tested it on iPad it worked

Reply:
do you want me to replace  UIApplication.getTopMostViewController()  with 
"(UIApplication.shared.delegate as? AppDelegate)?.window?.rootViewController?." ??

Reply:
I'm happy it works, I'm just thinking about edge case where `topVC` is nil, which means nothing happens and alert is not presented, right?